### DISCUSSION TITLE
Refreshing Browser forces authentication

### DISCUSSION THREAD

**Discussion by brettb33** [abacritt/angularx-social-login#667] [Q&A]
I've just upgraded my Angular 14 project from version 4.1.0 of angularx-social-login to the latest, 1.2.5.

I've noticed that in the old version you could refresh the browser and change the URL in the address bar and subscribing to the SocialAuthService.authState would return the logged in SocialUser object. In the latest version the SocialUser object returned is null and the authentication process is performed.

Can I ask if this is the expected behaviour now?

**Comment by flozi76** [comment]
Did you find a solution for that problem?

**Reply by brettb33** [reply]
No, I just presumed that this was the new behavior as it's more secure so probably not a problem at all.

**Comment by flozi76** [comment]
I found the solution of the problem: https://youtu.be/G4BBNq1tgwE
The problem is, that the library at the end is not compatible with the new API of Google. Check out the video it is wery well explained.
